### DISCUSSION TITLE
Dimensions change when saving

### DISCUSSION THREAD

**Discussion by LEDSAVE** [jgraph/drawio-desktop#1863] [Q&A]
Hi everyone.
When I save the [draw.io](http://draw.io/) as .png the dimensions change in the saved file.
When I open the .png in draw.io desktop the dimensions are shown correctly, but Windows explorer and other software shows the dimensions in the saved file differently to those draw.io shows.
Sometimes this can be by one point in each dimension, but now I have had one which keeps the width and changes the height from 40 to 89!
It's driving me INSANE!
Please help!
Thanks

**Comment by alderg** [comment]
Please attach the file.

**Reply by LEDSAVE** [reply]
![Screenshot 2024-09-10 120012](https://github.com/user-attachments/assets/f7119d04-b90f-479a-b3a2-462c84062705)
![Screenshot 2024-09-10 115834](https://github.com/user-attachments/assets/6c84056d-6686-4103-bc7f-c09bf753cbbf)
![Screenshot 2024-09-10 115754](https://github.com/user-attachments/assets/0e971684-a548-424c-b26f-d2bbdd5b00cd)
Thanks

**Reply by LEDSAVE** [reply]
![export 2](https://github.com/user-attachments/assets/9ba0fff7-ad99-4bb4-82f9-00753ae74774)

**Comment by davidjgraph** [comment]
And what's the draw.io version? This the window 64 bit version? What's version of Windows?

**Comment by LEDSAVE** [comment]
Hi David

Thanks for your reply.

Draw.io desktop is c24.7.8. Used draw.io-24.7.8-windows-installer.exe

My windows is 64 bit.

Even draw.io web saved slightly different sizes of file, but nothing like this!

Thanks


Kind regards,
David
